### PR TITLE
fix(AppHeader): visual focus to menu descriptions

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

before:
After opening "Catalog" menu focus is set to the "Popular course topics Explore free or paid courses in topics that interest you." text and focus indicator is not visible.
Other occurrences: Resources menu item.

after: 
After opening "Catalog" menu (or "Resources" menu) w/ keyboard, focus indicator should be visible.

### PR Checklist

- [ ] Related to designs:
- [X] Related to JIRA ticket: [DISC-899]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

In preview env,
using keyboard, navigate to catalog and open dropdown. confirm visible outline around description.
repeat with resources dropdown. 

Now repeat both steps using mouse only, confirm you dont see any outline upon opening the dropdown menu

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33370)  | [Monolith Env](https://pr-33370-monolith.dev-eks.codecademy.com/) | 
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/536)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-536)   | 

[DISC-899]: https://codecademy.atlassian.net/browse/DISC-899?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/gamut-labs/src/AppHeader/AppHeaderElements/AppHeaderCatalogSection/index.tsx [hunk 1]
```diff
@@ -113,17 +116,17 @@ export const AppHeaderCatalogSection = React.forwardRef<
     title: string;
     subtitle: string;
   }> = ({ title, subtitle }) => (
-    <FlexBox
+    <DescriptionSectionContainer
       data-focusablecatalog="true"
       data-testid="title-description-section"
-      tabIndex={-1}
+      tabIndex={0}
```
Comment (saghdaey) [inline packages/gamut-labs/src/AppHeader/AppHeaderElements/AppHeaderCatalogSection/index.tsx:122]:
```suggestion
      tabIndex={-1}
```

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.222-alpha.ba7519.0<br/>@codecademy/gamut-labs@33.6.13-alpha.ba7519.0<br/>@codecademy/styleguide@62.0.7-alpha.ba7519.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://636133e49f6c814ce6f67876--gamut-preview.netlify.app>https://636133e49f6c814ce6f67876--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/636133e49f6c814ce6f67876>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.223-alpha.ba1d43.0<br/>@codecademy/gamut-labs@33.6.14-alpha.ba1d43.0<br/>@codecademy/styleguide@62.1.1-alpha.ba1d43.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63615fd54caeca274e499d29--gamut-preview.netlify.app>https://63615fd54caeca274e499d29--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63615fd54caeca274e499d29>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63616d52e398653684347a8c--gamut-preview.netlify.app>https://63616d52e398653684347a8c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63616d52e398653684347a8c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.226-alpha.ffd23e.0<br/>@codecademy/gamut-labs@33.6.17-alpha.ffd23e.0<br/>@codecademy/styleguide@62.2.2-alpha.ffd23e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6363d7b07498e90089598f6e--gamut-preview.netlify.app>https://6363d7b07498e90089598f6e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6363d7b07498e90089598f6e>Deploy Logs</a></p>

File: packages/gamut-labs/src/AppHeader/shared.tsx [hunk 2]
```diff
@@ -88,3 +88,13 @@ export const StyledDropdown = styled(motion.div)(
 export const LayoutGridAntiAliased = styled(LayoutGrid)`
   -webkit-font-smoothing: antialiased;
 `;
+
+/* for Resources & Catalog menus */
+export const DescriptionSectionContainer = styled(FlexBox)(
+  css({
+    '&:focus-visible': {
```
Comment (saghdaey) [inline packages/gamut-labs/src/AppHeader/shared.tsx:95]: only overrides when user is navigating w/ keyboard

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.226-alpha.195311.0<br/>@codecademy/gamut-labs@33.6.17-alpha.195311.0<br/>@codecademy/styleguide@62.2.2-alpha.195311.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63642289f634ce18de3648ac--gamut-preview.netlify.app>https://63642289f634ce18de3648ac--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63642289f634ce18de3648ac>Deploy Logs</a></p>

**Review by cstatro — APPROVED** [review]
Nice this works great!

**Review by jakemhiller — APPROVED** [review]
lgtm!

**Pull request merged**